public/controllers/adminPanel.js

```diff
@@ -0,0 +1,28 @@
+var controller = angular.module('adminPanel',[]);
+controller.controller('admin', ['ConfigService', '$scope', '$http', 'Upload', '$routeParams',
```

Comment:
@Shujaat89 shouldn't we remove extra / unused dependency injections from controller declaration ?